src/pages/api/ngo.ts

```diff
@@ -1,9 +1,8 @@
 import { NextApiRequest, NextApiResponse } from 'next'
 import { AutoformResponseBody } from '../../types/api'
 import Fuse from 'fuse.js'
+import ngos from '../../../public/ngos.json'
```

Comment:
Toto by sme potom naozaj mohli spravit cez ten fetch nech to netaha kazdemu 2 mega jsonu

Reply:
Kod v `src/pages/api/*` je server-side takze to netaha cely json.